### PR TITLE
Add corelation between vertex count and resolution

### LINKED ISSUE #174: Increase vertex count for small drops.
Currently small drops have few vertices which looks unnatural.

marebllous-image (16)

### LINKED ISSUE #178: Body block should use full space
Should also avoid scrolling.

### PR DESCRIPTION
**Pull request by JBenda**
* for higher resolution we now use more vertices -> less artefacts
    * may a problem when dropping first on a very low scale and then
      switch to a high resolution
* add a canvasSize option to propergate this information
* fixes Debug build

close #174

### CODE REVIEW

File: backend/include/Options.hpp [hunk 1]
```diff
@@ -10,6 +10,7 @@ class Options
         Options();
         std::vector<Palette> Palettes; ///< All saved palettes.
         size_t activePalette; ///< Index of currently active palette inside palettes vector.
+		size_t canvasSize; ///< size of canvas in pixel.
```
Comment (t-erxleben) [inline backend/include/Options.hpp]: Already stored in WGLContext?

**Review by t-erxleben — CHANGES_REQUESTED** [review]
Nice work, but the canvas size is already stored in WGLContext. Is it necessary to duplicate this?

File: backend/include/Scene.hpp [hunk 2]
```diff
@@ -108,13 +108,18 @@ class Scene
...
 		if(displacements.empty()) { return; }
 
+		size_t vCount = 0;
 		for(auto& p : *this) {
-			p.displace(displacements);
+			p.displace(displacements, _canvasSize);
+			vCount += p.getVertCount();
 		}
 
+		std::cout << "vCount: " << vCount << "\n";
```
Comment (t-erxleben) [inline backend/include/Scene.hpp]: Could you remove the debug prints?

**Review by t-erxleben — CHANGES_REQUESTED** [review]

File: backend/include/Scene.hpp (hunk 2 above)
Comment (JBenda) [reply backend/include/Scene.hpp]: sure

File: backend/include/Options.hpp (hunk 1 above)
Comment (JBenda) [reply backend/include/Options.hpp]: Thanks for the hint ^^ Totally forgot that

**Review by t-erxleben — APPROVED** [review]
Thx!

**Pull request merged**